docs/ide/tutorial-csharp-aspnet-core.md

```diff
@@ -126,11 +126,11 @@ This solution follows the **Razor Page** design pattern. It is different than th
 
 1. Keep the browser window open and return to Visual Studio.
 
-1. In Visual Studio, choose **About.cshtml**. Then, delete the word _changed_ and in its place, add the words _file and directory_.
+1. In Visual Studio, choose **About.cshtml**. Then, delete the word _additional_ and in its place, add the words _file and directory_.
 
     ![Change the text in the About.cshtml file](../ide/media/csharp-aspnet-razor-aboutcshtml-mycoreapp-code-changed.png)
 
-1. Choose **About.cshtml.cs**. Then, clean up the `using` directives at the top of the file by using the following shortcut:
+1. Choose **About.cshtml.cs**. Then, clean up the `using` directives at the top of the file by using the following procedure:
 
```

Comment:
@jmayntzhusen I must also respectfully decline this edit. The information is an addendum to a step that's already in a procedure, and so we would not want to name this as a procedure within a procedure.